libraries/AP_Generator/AP_Generator_Loweheiser.h

```diff
@@ -0,0 +1,198 @@
+// -*- tab-width: 4; Mode: C++; c-basic-offset: 4; indent-tabs-mode: nil -*-
+#pragma once
+
+#include "AP_Generator_Backend.h"
+
+#if AP_GENERATOR_LOWEHEISER_ENABLED
+
+#include <AP_Common/AP_Common.h>
+#include <RC_Channel/RC_Channel.h>
+#include <GCS_MAVLink/GCS_MAVLink.h>
+
+#include <stdint.h>
+
+/*
+   Example setup:
+ param set SERIAL2_PROTOCOL 2  # Generator protocol
+ param set SERIAL2_BAUD 115200
+ param set GEN_TYPE 4
```

Comment:
Should this be added here with a note of why?
```suggestion
 param set SERIAL2_OPTIONS 1024 # Don't forward mavlink traffic to/from the generator
 param set GEN_TYPE 4
```